go/client/cmd_simplefs_close.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"encoding/hex"
+	"errors"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+// CmdSimpleFSClose is the 'simplefs srar' command.
+type CmdSimpleFSClose struct {
+	libkb.Contextified
+	opid keybase1.OpID
+}
+
+// NewCmdSimpleFSClose creates a new cli.Command.
+func NewCmdSimpleFSClose(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "close",
```

Comment:
I think we should have `keybase fs wait <opid>` and `keybase fs kill <opid>`. Shouldn't that cover all cases?